qiskit/pulse/commands/_sample_pulse.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Sample pulse.
+"""
+
+from qiskit.pulse.commands._pulse_command import PulseCommand
+
+
+class SamplePulse(PulseCommand):
+    """Container for functional pulse."""
+
+    def __init__(self, duration):
```

Comment:
The sample pulse should accept a second required argument `samples` which is a complex numpy array.